README.md

```diff
@@ -14,41 +14,42 @@ visit [Chat View Example](https://chat-view-8f1b5.web.app/#/).
 ![The example app running in iOS](https://raw.githubusercontent.com/SimformSolutionsPvtLtd/flutter_chat_ui/main/preview/chatview.gif)
 
 ## Migration guide for release 2.0.0
-Removed `showTypingIndicator` field from `ChatView` and replaced it with `ChatController.showTypingIndicator`.
 
-Before:
-```dart
-ChatView(
-  showTypingIndicator:false,
-),
-```
+- Updated `sendBy` field to `sentBy` in `Message` class.
 
-After:
-```dart
-/// use it with your [ChatController] instance.
-_chatContoller.setTypingIndicator = true; // for showing indicator
-_chatContoller.setTypingIndicator = false; // for hiding indicator
+- Updated `chatUsers` field to `otherChatUsers` in `ChatController` class.
 
-ChatView(
-  chatController: _chatController,
-),
-```
+- Removed `showTypingIndicator` field from `ChatView` and replaced it with `ChatController.showTypingIndicator`.
 
-Updated `sendBy` field to `sentBy` in `Message` class.
+  Before:
+    ```dart
+    ChatView(
+      showTypingIndicator:false,
+    ),
+    ```
 
-Before:
-```dart
-Message(
-  sendBy: '1',
-),
-```
+  After:
+    ```dart
+    /// use it with your [ChatController] instance.
+    _chatContoller.setTypingIndicator = true; // for showing indicator
+    _chatContoller.setTypingIndicator = false; // for hiding indicator
+    ```
 
-After:
-```dart
-Message(
-  sentBy: '1',
-),
-```
+- Moved `currentChatUser` field from `ChatView` widget to `ChatController` class.
```

Comment:
The variable's name from ChatView class was not currentChatUser. Can you please be more precise?